bundles/org.openhab.binding.epsonprojector/src/main/java/org/openhab/binding/epsonprojector/internal/EpsonProjectorHandler.java

```diff
@@ -0,0 +1,451 @@
+                    return new DecimalType(err);
+                case ERR_MESSAGE:
+                    String errString = remoteController.getErrorString();
+                    return new StringType(errString);
+                case FLESH_TEMP:
+                    int fleshColor = remoteController.getFleshColor();
+                    return new DecimalType(fleshColor);
+                case GAIN_BLUE:
+                    int gainBlue = remoteController.getGainBlue();
+                    return new DecimalType(gainBlue);
+                case GAIN_GREEN:
+                    int gainGreen = remoteController.getGainGreen();
+                    return new DecimalType(gainGreen);
+                case GAIN_RED:
+                    int gainRed = remoteController.getGainRed();
+                    return new DecimalType(gainRed);
+                case GAMMA:
+                    Gamma gamma = remoteController.getGamma();
+                    return new StringType(gamma.toString());
+                case GAMMA_STEP:
+                    logger.warn("Get '{}' not implemented!", commmandType.toString());
+                    return UnDefType.UNDEF;
+                case HKEYSTONE:
+                    int hKeystone = remoteController.getHorizontalKeystone();
+                    return new DecimalType(hKeystone);
+                case HPOSITION:
+                    int hPosition = remoteController.getHorizontalPosition();
+                    return new DecimalType(hPosition);
+                case HREVERSE:
+                    Switch hReverse = remoteController.getHorizontalReverse();
+                    return hReverse == Switch.ON ? OnOffType.ON : OnOffType.OFF;
+                case KEY_CODE:
+                    break;
+                case LAMP_TIME:
+                    int lampTime = remoteController.getLampTime();
+                    return new DecimalType(lampTime);
+                case LUMINANCE:
+                    Luminance luminance = remoteController.getLuminance();
+                    return new StringType(luminance.toString());
+                case MUTE:
+                    Switch mute = remoteController.getMute();
+                    return mute == Switch.ON ? OnOffType.ON : OnOffType.OFF;
+                case OFFSET_BLUE:
+                    int offsetBlue = remoteController.getOffsetBlue();
+                    return new DecimalType(offsetBlue);
+                case OFFSET_GREEN:
+                    int offsetGreen = remoteController.getOffsetGreen();
+                    return new DecimalType(offsetGreen);
+                case OFFSET_RED:
+                    int offsetRed = remoteController.getOffsetRed();
+                    return new DecimalType(offsetRed);
+                case POWER:
+                    PowerStatus powerStatus = remoteController.getPowerStatus();
+
+                    if (powerStatus == PowerStatus.ON) {
+                        return OnOffType.ON;
+                    } else {
```

Comment:
Maybe `return powerStatus == PowerStatus.ON ? OnOffType.ON : OnOffType.OFF;` optionally with a static import of ON/OFF